src/main/java/com/academy/ui/pages/header/HeaderComponent.java

```diff
@@ -27,4 +32,24 @@ public WebElement getGroupButton() {
     public void clickMenu() {
         this.getProfileMenuButton().click();
     }
-}
+
+    public WebElement getAddClubButton() {
+        if (addClubButton == null) {
+            addClubButton = rootElement.findElement(By.xpath("//button[contains(@class,'add-club-button')]"));
+        }
+        return addClubButton;
+    }
+
+    public void addClubButtonClick() {
+        sleep(10);
```

Comment:
Is it possible to use wait.until element is displayed instead of using the sleep method in this case?

Reply:
Oh, good catch, thank you, I used this sleep, only for debugging) I'll remove it)